trackhubs/tests/test_utils.py

```diff
@@ -0,0 +1,37 @@
+"""
+.. See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+"""
+
+import pytest
+import trackhubs.utils
+
+
+@pytest.mark.parametrize(
+    'the_string, expected_output, expected_type',
```

Comment:
Shall we add a couple of cases where the input is already a list or a dict?

Reply:
Added 👍